x/assets/keeper/genesis.go

```diff
@@ -0,0 +1,59 @@
+package keeper
+
+import (
+	"github.com/ExocoreNetwork/exocore/x/assets/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// InitGenesis initializes the module's state from a provided genesis state.
+func (k Keeper) InitGenesis(ctx sdk.Context, data *types.GenesisState) {
```

Comment:
Traversing and working with large amounts of data in InitGenesis (e.g. customer chain information, token information, and deposit information) may lead to performance issues, especially during initial block loading

Reply:
Added a `TODO` to this effect for later.